public-ui/src/app/data-browser/concept-charts/concept-charts.component.ts

```diff
@@ -18,7 +18,8 @@ export class ConceptChartsComponent implements OnChanges, OnInit, OnDestroy {
   @Input() concept: Concept;
   @Input() backgroundColor = '#ECF1F4'; // background color to pass to the chart component
   @Input() showSources = true;
-  @Input() showGenderGraph = false;
```

Comment:
Doesn't necessarily need to be in this PR, but would it ever make sense to show multiple of these at the same time? If not, a single property with an enum value e.g. `GraphType` would probably make way more sense here and could simplify a lot of this verbose code, e.g. ngSwitch statement.

Reply:
Added enum and made changes. Thanks